### ISSUE TITLE
Add transaction history to internal BTCPay wallet

### ISSUE THREAD

**Issue by bitcoinshirt** [btcpayserver/btcpayserver#211]
I have a suggestion for further improvement of the internal BTCPay wallet, so I'll leave it here for the future.

Can we add the transaction history. I’m finding myself in a situation where I use BTCPay internal wallet more than Ledger app, I just lack transaction history which would help me keep the track of refunds for example.

**Comment by NicolasDorier** [comment]
Yeah, this will be built when I will start working on exchange integration.
